tfjs-automl/package.json

```diff
@@ -27,17 +27,18 @@
   },
   "devDependencies": {
     "@babel/polyfill": "^7.8.7",
+    "@tensorflow/tfjs-backend-webgl": "^3.9.0",
     "@tensorflow/tfjs-converter": "^3.9.0",
     "@tensorflow/tfjs-core": "^3.9.0",
-    "@tensorflow/tfjs-backend-webgl": "^3.9.0",
     "@types/jasmine": "~2.8.6",
     "clang-format": "~1.8.0",
+    "jasmine": "3.1.0",
```

Comment:
jasmine was missing as a dependency of tfjs-automl, so it was using the root jasmine version. This caused tests to fail when the root jasmine was upgraded to 4.